crates/wash-cli/Cargo.toml

```diff
@@ -67,7 +67,7 @@ warp = { workspace = true }
 warp-embed = { workspace = true }
 wascap = { workspace = true }
 wash-lib = { workspace = true, features = ["cli", "parser", "nats", "start"] }
-wasmcloud-control-interface = { workspace = true }
+wasmcloud-control-interface = "0.33"
```

Comment:
These are now going to stay pinned to a released version until we're ready to update wash to use a released version of a host

Reply:
Pointing out what may be already obvious, this means that the released version of wash won't be able to interface with the host as-is in `main`. I think that's fine for now 🤔